packages/aragon-wrapper/src/index.js

```diff
@@ -894,6 +889,12 @@ export default class Aragon {
       } catch (_) { }
     }
 
+    let forwarders = await this.forwarders.take(1).toPromise().then(
```

Comment:
What do you think about putting all of this bit into `calculateForwardingPath()`, so that `calculateTransactionPath()` just passes the `finalForwarder`?

It seems like anything forwarder-related should be in `calculateForwardingPath()`, and allow us to remove some of this duplicate logic.

Reply:
Calculating the `forwardersWithPermission` when a `finalForwarder` isn't provided requires the `methodName` to validate permissions. `methodName` isn't (and shouldn't be) passed to `calculateForwardingPath()`.

I also think it makes sense for this to be here because the final forward is the transaction itself, which isn't part of the forwarding path.